### PR TITLE
Modify devices

### PR DESCRIPTION
**Pull request by zhangyihui1**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -11,19 +11,18 @@ OWT Android requires using Android SDK API level 16 or above. In order to use ha
 The following devices have been validated with our SDK:
 
    + Huawei* Mate9 (Android 7.0)
+   + Huawei* Mate20 (Android 9.0)
```
Comment (jianjunz) [inline docs/index.md]: Please keep this list in alphabet order, i.e.: put mate 20 before meta 9 because 2 is less than 9.

File: docs/index.md [hunk 2]
```diff
@@ -11,19 +11,18 @@ OWT Android requires using Android SDK API level 16 or above. In order to use ha
...
    + Huawei* NXT-AL10 (Android 6.0)
    + Huawei* P8 (Android 5.0)
+   + Huawei* P20 (Android 8.1)
```
Comment (jianjunz) [inline docs/index.md]: ditto.

File: docs/index.md (hunk 1 above)
Comment (zhangyihui1) [reply docs/index.md]: done

File: docs/index.md (hunk 2 above)
Comment (zhangyihui1) [reply docs/index.md]: done

**Pull request merged**